bigframes/ml/llm.py

```diff
@@ -307,14 +307,10 @@ def predict(
 
         (X,) = utils.convert_to_dataframe(X)
 
-        if len(X.columns) != 1:
-            raise ValueError(
-                f"Only support one column as input. {constants.FEEDBACK_LINK}"
-            )
-
-        # BQML identified the column by name
-        col_label = cast(blocks.Label, X.columns[0])
-        X = X.rename(columns={col_label: "prompt"})
+        if len(X.columns) == 1:
```

Comment:
I think we should make another check in the else clause - that the multi-column input does have a "prompt" column. Also add test for that scenario

Reply:
@tswast had a suggestion that we shouldn't do much client side checks. I'm trying to follow: if the error message is meaningful to the user, then rely on server side checks. Otherwise we have to wrap server error messages or return client side error messages.